SPECS/golang/golang.spec

```diff
@@ -145,6 +143,10 @@ fi
 %{_bindir}/*
 
 %changelog
+* Mon Oct 16 2023 Nan Liu <liunan@microsoft.com> - 1.20.9-1
+- Bump version to 1.20.9 to address CVE-2023-29409, CVE-2023-39318, CVE-2023-39319, CVE-2023-39323, CVE-2023-39533
+- Remove patch that no longer applies
```

Comment:
It doesn't apply, because it's already part of the new version or there's a conflict when the patch is applied?

Reply:
The removed patch is already in the new version.